### PR TITLE
add support for configuration files

### PR DESCRIPTION
**Pull request by mreitinger**

### CODE REVIEW

File: lib/WWW/WTF/Testcase/Config.pm [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package WWW::WTF::Testcase::Config;
+use Moose;
+use File::Util;
```
Comment (christophhalbi) [inline lib/WWW/WTF/Testcase/Config.pm]: please add to Makefile (all three)

File: lib/WWW/WTF/Testcase/Config.pm [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    lazy    => 1,
+    default => sub { {}; },
+);
+
+sub _extend_config_from_directory {
+    my ($self, $path) = @_;
+
+    my @config_files = ();
+
+    File::Util->new->list_dir(
```
Comment (christophhalbi) [inline lib/WWW/WTF/Testcase/Config.pm]: maybe we can use a unified way for searching files (I use File::Find::Rule in lib/WWW/WTF/HTTPResource/Types/PDF.pm)

**Comment by mreitinger** [comment]
extended the Makefile and migrated to File::Find::Rule

**Review by christophhalbi — APPROVED** [review]
ACK

**Pull request merged**